joomla/joomla-cms
# Duplicate String and not to be deprecated.

Somehow we ended up with a dupicate language string, the second one having a deprecation note.

### Summary of Changes
Since the string is still used both in J3 and J4, this PR removes the second appearance together with the note.


### Testing Instructions
The button where this text is used isn't visible by default. It gets hidden by JS once the drag area in the extension installer view is loaded. I guess it would be visible if JS is disabled or broken or whatever.
You can see it in the source code thought.

### Expected result
Text is there.


### Actual result
Text is there, but string is doubled in the INI file


### Documentation Changes Required
None
